modules/artifact-activation/src/test/resources/apps/basic/mule-artifact.json

```diff
@@ -3,7 +3,13 @@
   "minMuleVersion": "4.4.0",
   "requiredProduct": "MULE",
   "classLoaderModelLoaderDescriptor": {
-    "id": "mule"
+    "id": "mule",
+    "attributes": {
+      "exportedResources": [
```

Comment:
why is it necessary to add this when it wasn't before? What will happen when an app packaged without this is deployed here?

Reply:
It should have been there from the beginning. At first when working in the descriptors API for the framework we were exporting everything unless stated otherwise here in the `mule-artifact.json`, but that's not Mule's current behaviour. It's the opposite, everything should be treated as a local package/resource unless explicitly indicated in this file.